extendr-api/Cargo.toml

```diff
@@ -34,6 +34,11 @@ rstest = "0.16.0"
 [features]
 default = []
 
+# This dummy feature enables all features that increase the functionality of 
+# extendr, via conversions or R features. Features that change behaviour
+# but do not add functionality (such as `libR-sys/use-bindgen`) are excluded
+full-functionality = ["graphics", "either", "ndarray", "num-complex", "serde"]
```

Comment:
I think this should be `full-support`, no? Since you reference `--features full-support` in CI script

Reply:
I asked Mossa to rename it to `full-functionality`. I just missed updating it in the CI script.